### DISCUSSION TITLE
Create multiple notes at the same time?

### DISCUSSION THREAD

**Discussion by Fierce-Ninja** [SilentVoid13/Templater#412] [Help]
I am trying to create multiple notes from multiple words using a single template.  I would use <% tp.file.title %> to put a heading in the note based on the word it is being created from.  So the word would be <% tp.file.title %>

So let's say I have the following words with dangling links (i.e., not yet a note yet but just brackets to indicate they would be notes):

[[Chamomile]]
[[Blueberry]]

Let's consider a simple template:

```
<% tp.file.title %>
This is the first line

```

I am wishing that I can run some kind of a batch to trigger the above template for the above mentioned notes at the same time **without** me having to manually do this operation twice.

So at the end of this operation, 

the first note would look like this

# Chamomile
This is the first line

And the usecond note would look like this

# Blueberry
This is the first line

Can anyone please tell me if above is possible. How?

**Comment by AB1908** [comment]
You'd need to write a little JS to do this.
